src/angular/datepicker/datepicker-content/datepicker-content.html

```diff
@@ -1,5 +1,9 @@
 <sbb-calendar
   cdkTrapFocus
+  role="dialog"
+  [attr.aria-modal]="true"
```

Comment:
Is this intentionally not `aria-modal="true"`?

Reply:
hmm copied from material